docs/config/config.xml

```diff
@@ -61,27 +61,48 @@
         <!-- defines how visit labels are assigned -->
         <visitLabel subprojectID="1">
             <labelSet>
-                <item value="V01">V01</item>
-                <item value="V02">V02</item>
+                <item value="V1">V1</item>
+                <item value="V2">V2</item>
+                <item value="V3">V3</item>
             </labelSet>
         </visitLabel>
         <visitLabel subprojectID="2">
             <labelSet>
-                <item value="V01">V01</item>
-                <item value="V02">V02</item>
+                <item value="V1">V1</item>
+                <item value="V2">V2</item>
+                <item value="V3">V3</item>
+            </labelSet>
+        </visitLabel>
+        <visitLabel subprojectID="3">
+            <labelSet>
+                <item value="V3">V3</item>
+                <item value="V4">V4</item>
+                <item value="V5">V5</item>
+                <item value="V6">V6</item>
+            </labelSet>
+        </visitLabel>
+        <visitLabel subprojectID="4">
+            <labelSet>
+                <item value="V3">V3</item>
+                <item value="V4">V4</item>
+                <item value="V5">V5</item>
+                <item value="V6">V6</item>
             </labelSet>
         </visitLabel>
 
-         <!-- certification module -->
+        <!-- certification module -->
         <Certification>
-            <EnableCertification>0</EnableCertification>
+            <EnableCertification>1</EnableCertification>
```

Comment:
I don't think this should be the default for LORIS, it makes it harder to set things up on a new install